dockers/docker-fpm-frr/staticd.conf.j2

```diff
@@ -0,0 +1,29 @@
+!
+{% block banner %}
+! =========== Managed by sonic-cfggen DO NOT edit manually! ====================
+! generated by templates/frr/staticd.conf.j2 using config DB data
+! file: staticd.conf
+!
+{% endblock banner %}
+!
+{% block sys_init %}
+hostname {{ DEVICE_METADATA['localhost']['hostname'] }}
+password zebra
+enable password zebra
+{% endblock sys_init %}
+!
+{% block default_route %}
+! set static default route to mgmt gateway as a backup to learned default
+{% for (name, prefix) in MGMT_INTERFACE|pfx_filter %}
+{% if prefix | ipv4 %}
+ip route 0.0.0.0/0 {{ MGMT_INTERFACE[(name, prefix)]['gwaddr'] }} 200
+{% endif %}
+{% endfor %}
+{% endblock default_route %}
+!
+{% block logging %}
+log syslog informational
+log facility local4
+{% endblock logging %}
+!
```

Comment:
The same for block logging. 
Is it supported by staticd?

Reply:
Same as the comment above. By the way, I think the logging block should be moved to the place below enable password option for consistency with the original conf files generated by FRR. And zebra.conf.j2 also need to do the same modification. What do you think?

Reply:
I think let's keep the options in the current order. Thank you for your comments.